docs/table_design/Data_distribution.md

```diff
@@ -142,27 +148,75 @@ CREATE TABLE site_access
     pv BIGINT SUM DEFAULT '0'
 )
 AGGREGATE KEY(site_id, city_code, user_name)
-DISTRIBUTED BY HASH(site_id,city_code) BUCKETS 10;
+DISTRIBUTED BY HASH(site_id,city_code);
 ```
 
-### Determine the number of tablets
+### Determine the number of buckets
 
-Tablets reflect how data files are organized in StarRocks. Since StarRocks 2.5, you do not need to set the number of buckets when a table is created, and StarRocks sets the number of buckets automatically.
+Buckets reflect how data files are organized in StarRocks.
 
-```SQL
-CREATE TABLE site_access(
-    site_id INT DEFAULT '10',
-    city_code SMALLINT,
-    user_name VARCHAR(32) DEFAULT '',
-    pv BIGINT SUM DEFAULT '0'
-)
-AGGREGATE KEY(site_id, city_code, user_name)
-DISTRIBUTED BY HASH(site_id,city_code); --do not need to set the number of buckets
-```
+- How to set the number of buckets when a table is created
+
+  - Method 1: automatically set the number of buckets (Recommended)
+
+    Since v2.5.7, StarRocks supports automatically setting the number of buckets based on machine resources and data volume for a partition.
+
+    Example:
+
+    ```SQL
+    CREATE TABLE site_access(
+        site_id INT DEFAULT '10',
+        city_code SMALLINT,
+        user_name VARCHAR(32) DEFAULT '',
+        pv BIGINT SUM DEFAULT '0'
+    )
+    AGGREGATE KEY(site_id, city_code, user_name)
+    DISTRIBUTED BY HASH(site_id,city_code); -- do not need to set the number of buckets
+    ```
+
+    To enable this feature, make sure that the FE dynamic parameter `enable_auto_tablet_distribution` is set to true. After a table is created, you can execute [SHOW CREATE TABLE](../sql-reference/sql-statements/data-manipulation/SHOW%20CREATE%20TABLE.md) to view the bucket number automatically set by StarRocks.
+
+  - Method 2: manually set the number of buckets
+
+    If you intend to set the number of buckets, StarRocks 2.4 and later versions support using multiple threads to scan a tablet in parallel during a query, thereby reducing the dependency of scanning performance on the tablet count. We recommend that each tablet contains about 10 GB of raw data. You can estimate the amount of data in each partition of a table and then decide the number of tablets. To enable the parallel scanning on tablets, make sure the `GLOBAL enable_tablet_internal_parallel` is enabled.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/487e7fc2-7f7f-475e-8e03-293472b3d56b)
I think we should use "contain" rather than "contains" here